### PR TITLE
Cg ss display map

### PR DESCRIPTION
**Pull request by snsebro**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description
-Installed google-map-react library
-Created config file with API-KEY and added to gitignore gile
-Created google-map-react component
-Rendered map in App component with default location and zoom props

## Related Issue

Story: Display a Basic Map

## Acceptance Criteria

- [x] google-map-react is added to the project dependencies
- [x] A Map component is created that renders a default Google Map at a hardcoded lat/long & hardcoded zoom level
- [x] The map should render in production, not development mode using the following API key:

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|     | :scroll: Docs              |

### After

image

## Testing Steps / QA Criteria

From home screen (path = '/') a map should render with a set location. In the App component if you update the lat and lng properties in the location object the map should update to that location.

### CODE REVIEW

**Comment by rachel-fischoff** [comment]
looks good! I updated it with my long & lat for my town (durham, NC) & played with the different zoom levels. I enjoyed having zoom levels at 14 & 15 to see the streets but I also like that the map is responsive!

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# config.js
```
Comment (segdeha) [inline .gitignore:26]: If this is no longer needed, please delete. Thanks!

File: package.json [hunk 2]
```diff
@@ -52,7 +53,10 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx}": ["eslint --fix", "prettier --write"],
+    "*.{js,jsx}": [
```
Comment (segdeha) [inline package.json:56]: `prettier` making our `prettier` config code prettier 😂

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,5 @@
...
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+  height: 100vh;
+  width: 100%;
```
Comment (segdeha) [inline src/App.css:4]: Curious why you used different units for these 2 declarations? Did `100vw` not work for some reason?

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
+import Map from './components/Map';
 import './App.css';
 
 function App() {
+  const location = {
+    lat: 21.694,
+    lng: 71.7979,
```
Comment (segdeha) [inline src/App.js:7]: Interesting choice of location! How did you choose it?

File: src/App.js [hunk 5]
```diff
@@ -1,23 +1,17 @@
...
+  };
+
+  const zoomLevel = 8;
```
Comment (segdeha) [inline src/App.js:10]: We may need to adjust this once we have pins rendering, but it’s a good guess as a starting point!

File: src/App.js [hunk 6]
```diff
@@ -1,23 +1,17 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Map location={location} zoomLevel={zoomLevel} />
```
Comment (segdeha) [inline src/App.js:14]: 👍

File: src/components/Map.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
+import { API_KEY } from '../config.js';
```
Comment (segdeha) [inline src/components/Map.js:3]: Smart pulling this key out into a config file! 👏👏👏

**Review by segdeha — APPROVED** [review]
OK, I couldn’t help myself and did my review, anyway, before the other pair had a chance to review 😅 

@DeeThompson & @rachel-fischoff Please do still add your reviews so you’re familiar with the code here. It will also make it easier to do the 2nd merge. Thanks!

**Review by rachel-fischoff — APPROVED** [review]
Looks great! I added the lat/long for durham, nc. Only thing is maybe we could start at a higher zoom (12?)

File: src/App.js (hunk 5 above)
Comment (greffly) [reply src/App.js:10]: Yeah it's zoomed out fairly far, but we can adjust as we go.

**Comment by DeeThompson** [comment]
I looked over everything, I honestly have nothing to say but really good job! This is a fun project can't wait to keep developing it.

**Pull request merged**